lib/dotcom/utils/date_time.ex

```diff
@@ -0,0 +1,77 @@
+defmodule Dotcom.Utils.DateTime do
+  @moduledoc """
+  A collection of functions for working with date_times.
+
+  Consuming modules are responsible for parsing or converting date_times.
+  They should *always* call `coerce_ambiguous_date_time/1` before using a date_time.
+  This is mainly because Timex has so many functions that serve as entry points to date_times.
+  Those functions can return ambiguous date_times during DST transitions.
+  """
+
+  use Timex
+
+  alias Dotcom.Utils.DateTime.Behaviour
+
+  @behaviour Behaviour
+
+  @typedoc """
+  A date_time_range is a tuple of two date_times: {start, stop}.
+  Either the start or stop can be nil, but not both.
+  """
+  @type date_time_range() ::
+          {DateTime.t(), DateTime.t()} | {nil, DateTime.t()} | {DateTime.t(), nil}
+
+  @timezone Application.compile_env!(:dotcom, :timezone)
+
+  @doc """
+  Get the date_time in the set @timezone.
+  """
+  @impl Behaviour
+  def now(), do: Timex.now(@timezone)
+
+  @doc """
+  In the default case, we'll return a DateTime when given one.
+
+  Timex can give us ambiguous times when we "fall-back" in DST transitions.
+  That is because the same hour occurs twice.
+  In that case, we choose the later time.
+
+  Timex will return an error if the time occurs when we "spring-forward" in DST transitions.
+  That is because one hour does not occur--02:00:00am to 03:00:00am.
+  In that case, we set the time to 03:00:00am.
+
+  If we are given something tha tis not a DateTime, AmbiguousDateTime, or an error tuple, we log the input and return `now`.
```

Comment:
(Non-blocking): It looks like the `now` fallback was removed (thank you 🙂), so unless I missed something, I think we can also drop this part of the doc comment?

Reply:
Also a small nit, but if I _didn't_ miss something then here's a tiny spelling fix. 

```suggestion
  If we are given something that is not a DateTime, AmbiguousDateTime, or an error tuple, we log the input and return `now`.
```